functions/lib/integration/parsers/product-to-ecomplus.js

```diff
@@ -191,7 +191,7 @@ module.exports = (blingProduct, variations, storeId, auth, isNew = true, appData
       product.min_quantity = Number(blingProduct.itensPorCaixa)
     }
     if (blingProduct.tributacao && blingProduct.tributacao.ncm) {
-      product.mpn = [String(blingProduct.tributacao.ncm)]
+      product.mpn = String(blingProduct.tributacao.ncm)
```

Comment:
```suggestion
      product.mpn = [String(blingProduct.tributacao.ncm)]
```

No produto é um array de strings, o erro é só na variação (como você pode ver na resposta da API)